dev-util/tartos/tartos-1.0.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="TAR to shell script"
+DESCRIPTION="The tartos script is a standalone bash shell script which binds \
+different type of TAR archives to a single shell script (bash). This kind of \
+approach is generally adopted to make a single install scipt from an existing \
+tar archives with executable or installable software. It can be used to make \
+a self extracting archive/installer which does some additional jobs like \
+installing softwares, compared to the conventional tar archive files. The \
+tartos script itself is written in bash and thus depends on it and on the \
+package tar."
+HOMEPAGE="https://github.com/neurobin/tartos"
+COPYRIGHT="2015-2017 Md Jahidul Hamid"
```

Comment:
I can't find any copyright statement in the source archive. Does this mean the script is uncopyrighted?

Reply:
Not really a Copyright included, only thing popping up is 2015 and author in the man page ...